tests/testsuite/build_script.rs

```diff
@@ -2338,6 +2338,7 @@ fn rebuild_only_on_explicit_paths() {
     p.cargo("build -v")
         .with_stderr(
             "\
+warning: \"cargo:rerun-if-changed\" file \"foo\" not found.
```

Comment:
There is a `[WARNING]` macro that is normally used instead of saying `warning:`.

Reply:
Thanks, updated.